client/src/pages/Home.js

```diff
@@ -5,41 +5,41 @@ import "./Home.css";
 import ModalVideo from "../components/ModalVideo";
 import CityIcon from "./CityIcon";
 import DotMap from "../components/DotMap";
+import Hero from "../components/Hero";
 
 export function Home() {
-
 	const [show, setShow] = useState(false);
 	const [selectedIcon, setSelectedIcon] = useState(null);
 
-    const [rhythms, setRhythms] = useState([]);
-
-		const API_URL = "/api/rhythm_codes";
+	const [rhythms, setRhythms] = useState([]);
 
-		useEffect(() => {
-			fetch(API_URL)
-				.then((response) => {
-					return response.json();
-				})
-				.then((data) => {
-					console.log("Fetched Data:", data);
-					setRhythms(data);
-				})
-				.catch((error) => {
-					console.error("Error fetching data:", error);
-				});
-		}, []);
+	const API_URL = "/api/rhythm_codes";
 
+	useEffect(() => {
```

Comment:
These formatting changes are likely to cause conflicts with @christina-mifsud 's  major overhaul of `Home.js`

Please coordinate on release - I think this PR should be merged after #53